src/pages/a-complete-guide-to-useeffect/index.zh-hant.md

```diff
@@ -6,19 +6,19 @@ spoiler: Effect 是你資料流的一部分。
 
 你用 [Hooks](https://reactjs.org/docs/hooks-intro.html) 寫了一些元件。或許也寫了一個小的應用程式。你感到很滿足。你對它的 API 感到滿意而且在過程中學習了一些技巧。你甚至做了一些[客製化的 Hooks](https://reactjs.org/docs/hooks-custom.html) 來抽出重複的邏輯（移除 300 行程式碼！）然後跟你的同事炫耀。他們說「做得好」。
 
-但有時候，當你使用 `useEffect` 時，那些片段感覺不太契合。你有一種好像你遺漏了什麼的感覺。它看起來像是 class 的生命週期... 但它真的是嗎？你發現你問自己像是以下的某些問題：
+但有時候，當你使用 `useEffect` 時，某些程式碼片段感覺不太契合。你有一種好像你遺漏了什麼的感覺。它看起來像是 class 的生命週期... 但它真的是嗎？你發覺自己總是想著以下幾個問題：
 
 * 🤔 我要怎麼用 `useEffect` 複製 `componentDidMount` 的行為？
 * 🤔 我該怎麼正確的在 `useEffect` 裡拿到資料？`[]` 是什麼？
-* 🤔 我應該要把用到的函示指定成 effect 的依屬 (dependencies) 嗎？
-* 🤔 為什麼我有時候會進入重複拿資料的無窮迴圈？
+* 🤔 我應該要把用到的函式指定成 effect 的依屬 (dependencies) 嗎？
+* 🤔 為什麼我有時候會陷入重複拿取資料的無窮迴圈？
```

Comment:
Maybe we should also modify L55 to make it consistent.

Reply:
OK, I think so!